physics/solar_system.hpp

```diff
@@ -32,11 +35,13 @@ class SolarSystem final {
   // format for SolarSystemFile protocol buffers.
   SolarSystem(
       std::experimental::filesystem::path const& gravity_model_filename,
-      std::experimental::filesystem::path const& initial_state_filename);
+      std::experimental::filesystem::path const& initial_state_filename,
+      bool ignore_frame = false);
```

Comment:
I'd prefer to avoid the defaulted parameter by having both a 2- and a 3-argument constructor, one delegating to the other.

Reply:
As discussed, leaving as-is.